### PR TITLE
Create Clinch Strike Features: Issue #19

### PR DESCRIPTION
**Pull request by aryang13**

### CODE REVIEW

**Review by ivan-guo — APPROVED** [review]
lgtm, except some of the lines are a bit long such as line 363, maybe use \ and separate into new lines to improve readability

File: clinch_strike_feature.py [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+        Returns:
+        - (float, float): The clinch strikes for each fighter as a tuple
+        """
+        all_prev_fights = df.loc[:index-1]
+        
+        if not all_prev_fights.empty:
+            fighter_a_id_vals = all_prev_fights.fighter_a_id.values
+            fighter_b_id_vals = all_prev_fights.fighter_b_id.values
+
+            fighter_a_fights = all_prev_fights[(fighter_a_id_vals == fighter_a_id)][-last_fights:]
```
Comment (ivan-guo) [inline clinch_strike_feature.py:254]: Same thing as the other PR - should be this

`fighter_a_fights = all_prev_fights[(fighter_a_id_vals == fighter_a_id) | (fighter_b_id_vals == fighter_a_id)][-last_fights:]`

**Review by ivan-guo — CHANGES_REQUESTED** [review]

File: clinch_strike_feature.py (hunk 1 above)
Comment (aryang13) [reply clinch_strike_feature.py]: done!

**Review by ivan-guo — APPROVED** [review]

**Review by straw-hat-gupta — APPROVED** [review]
lgtm

**Pull request merged**